### PR TITLE
chore: Create rest api for rp adapter

### LINKED ISSUE #9: Create rest api for rp adapter

### PR DESCRIPTION
**Pull request by fqutishat**
closes #9

Signed-off-by: Firas Qutishat <firas.qutishat@securekey.com>

### CODE REVIEW

File: cmd/adapter-rest/startcmd/start.go [hunk 1]
```diff
@@ -60,14 +60,8 @@ const (
...
-	healthCheckEndpoint                = "/healthcheck"
-	hydraLoginEndpoint                 = "/login"
-	hydraConsentEndpoint               = "/consent"
-	oidcCallbackEndpoint               = "/callback"
-	createPresentationRequestEndpoint  = "/presentations/create"
-	handlePresentationResponseEndpoint = "/presentations/handleResponse"
-	userInfoEndpoint                   = "/userinfo"
-	uiEndpoint                         = "/ui"
+	healthCheckEndpoint = "/healthcheck"
+	uiEndpoint          = "/ui"
```
Comment (llorllale) [inline cmd/adapter-rest/startcmd/start.go:64]: why isn't this one under `pkg`?

Comment (fqutishat) [reply cmd/adapter-rest/startcmd/start.go:64]: I was thinking of moving healthCheckEndpoint

Comment (fqutishat) [reply cmd/adapter-rest/startcmd/start.go]: I moved healthCheckEndpoint

File: pkg/restapi/rp/operation/operations.go [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+	"time"
+
+	log "github.com/sirupsen/logrus"
+
+	"github.com/trustbloc/edge-adapter/pkg/internal/common/support"
+)
+
+// API endpoints.
+const (
+	hydraLoginEndpoint                 = "/login"
```
Comment (talwinder50) [inline pkg/restapi/rp/operation/operations.go:22]: Are these the same endpoint as in edge-sandbox hydra login and consent. I mean does this impact sandbox ?

Comment (fqutishat) [reply pkg/restapi/rp/operation/operations.go:22]: yes

Comment (llorllale) [reply pkg/restapi/rp/operation/operations.go:22]: @talwinder50 @fqutishat 

The design currently doesn't assume the same hydra instance. Connecting it to the same hydra instance requires additional work.

**Pull request merged**